pkg/config/jobs.go

```diff
@@ -282,6 +295,8 @@ type Periodic struct {
 	// Tags for config entries
 	Tags []string `json:"tags,omitempty"`
 
+	RetriggerFailed *RetriggerFailed `json:"retrigger_failed_run,omitempty"`
```

Comment:
```suggestion
	RetriggerFailed *RetriggerFailed `json:"retrigger_failed,omitempty"`
```

For consistency and to avoid verbose configuration 

TBH, I would actually prefer to call this a `Retry` instead of `Retrigger` (pretty much everywhere where you use `Retrigger`) The word "retry" even implies the `Failed` and it would read rather nicely in the config:

```
retry:
  attempts: 3
  run_all: true
```